homeassistant/components/sensor/speedtest.py

```diff
@@ -10,11 +10,13 @@
 from subprocess import check_output
 
 import homeassistant.util.dt as dt_util
+from homeassistant.components import recorder
 from homeassistant.components.sensor import DOMAIN
 from homeassistant.helpers.entity import Entity
 from homeassistant.helpers.event import track_time_change
 
 REQUIREMENTS = ['speedtest-cli==0.3.4']
+DEPENDENCIES = ['recorder']
```

Comment:
Speedtest sensor will now force the loading of recorder, even if user does not want it to be loaded